markdown/chapters/entombsday/deck-construction.md

```diff
@@ -0,0 +1,56 @@
+---
+authors: drynne
+order: 2
+title: Deck Construction
+---
+
+## Table of Contents
+
+## Preamble
+
+As previously stated, the shell of both Tin Fins and Doomsday are remarkably similar and the combo packages are roughly the same size. Because of this overlap, there are very few concessions made in deckbuilding.
+
+::decklist{path=2023/03/entombsday.drynne}
+::decklist{path=2023/03/entombsday.Nitro_Rick}
+
+### Notable Inclusions
```

Comment:
For this section overall I think you need to talk more about the components you need and what each set of tools does. Then discuss the decision making. 

Here we have a list of cards some of which have some useful info, others less so.